src/main/java/com/mraof/minestuck/data/AspectTreeBlocksData.java

```diff
@@ -1298,6 +1325,9 @@ public static void addToBlockTags(MinestuckBlockTagsProvider provider)
 		provider.tag(BlockTags.LEAVES).addTag(MSTags.Blocks.ASPECT_LEAVES);
 		provider.tag(BlockTags.SAPLINGS).addTag(MSTags.Blocks.ASPECT_SAPLINGS);
 		provider.tag(BlockTags.PLANKS).addTag(MSTags.Blocks.ASPECT_PLANKS);
+		provider.tag(BlockTags.WOODEN_SLABS).add(BLOOD_ASPECT_SLAB.get(), BREATH_ASPECT_SLAB.get(), DOOM_ASPECT_SLAB.get(), HEART_ASPECT_SLAB.get(),
+				HOPE_ASPECT_SLAB.get(), LIFE_ASPECT_SLAB.get(), LIGHT_ASPECT_SLAB.get(), MIND_ASPECT_SLAB.get(),
+				RAGE_ASPECT_SLAB.get(), SPACE_ASPECT_SLAB.get(), TIME_ASPECT_SLAB.get(), VOID_ASPECT_SLAB.get());
```

Comment:
```suggestion
		provider.tag(BlockTags.WOODEN_SLABS).addTag(MSTags.Blocks.ASPECT_SLABS);
```

Reply:
I swear I tried that at the start and it didn't work, but ofc now it works fine. 😩 No clue what I did wrong. Updated.